modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1177 @@
+
+In {productname} 6.4, all links in a `contenteditable="false"` now behave as expected.
+
+=== Inline boundary was rendered for noneditable inline boundary elements
+// TINY-9471
+
+Inline boundaries for `contenteditable="false"` elements were rendering styling in previous versions of {productname} 6.4 after the user `selected` before and after the `contenteditable="false"` elements node. The expected behaviour is that `contenteditable="false"` elements should *not* have inline boundaries when a caret is inside an `contenteditable="false"` anchor or root.
+
+Added a predicate check for inline boundaries before they render, and additional tests for `selections` for some special cases surrounding particular `contenteditable="false"` elements.
+
+In {productname} 6.4, it is now not possible to highlight inline boundary elements that are classified as a `contenteditable="false"` content element.
+
+=== Clicking on a disabled split button will no longer call the onAction callback
+// TINY-9504
+
+A bug was found in versions of {productname} prior to the 6.4 release, that would call the `onAction` callback for *split buttons* regardless of the action being `disabled`. This meant it was possible for the user to emit the `onAction` call wether or not the `disable` function was enabled or not.
+
+As a fix to the issue, {productname} 6.4 introduced a new conditional check to identify *if* the button is `enabled` or `disabled`. If the button is set to `enabled` then {productname} executes the secondary check, which will identify wether or not the button is a *split button*.  Split buttons are classified as “special buttons”,  as they are unique and require both `advlists` and `dropdowns` to function, this makes them more complex than traditional buttons.
+
+In {productname} 6.4, clicking on a disabled split button won't call the `onAction` callback anymore.
+
+=== The "Edit Link" dialog incorrectly retrieved the URL value when opened immediately after the link insertion
+// TINY-7993
+
+A bug was found within the context menu and dialog when inserting `links` in previous versions of {productname}. The issue resulted in the `open link` being deactivated within the context menu and dialog when the user would execute a specific chain of clicks. Such as:
+
+.Example
+====
+. Opening the editor
+. Type “test1 test2 test3” into the editor
+. Double click to select “test2”
+. Use the context menu to insert a new link over the text “test2”
+. Add a valid URL
+. Press `Enter`
+. Without taking any other action, right click on “test2”
+
+In this example the bug would deactivate the `open link` button within the context menu and dialog, which removes the users ability to reopen the “open link” dialog.
+====
+
+The fix now allows all three link-plugins to work within the context menu items.
```

Comment:
What are the three link plugins?

Reply:
* Link...
* Remove link
* Open link - previously this was deactivated